.github/workflows/build.yml

```diff
@@ -63,8 +69,17 @@ jobs:
       run: |
         coverage run -m pytest
         bash <(curl -s https://codecov.io/bash)
+
+  package-publish:
+    needs: coverage-lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
why not v3?

Reply:
didn't mean to mix too many changes in here... could be done in #104 once this here is merged

Reply:
now I did it anyway, and renamed the PR